CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+* Standardize notes titles in `carnival` and `cosmos`
```

Comment:
should this changelog note say "carnival" since no changes were made in cosmos in this PR?

Reply:
There is a change for one book from Cosmos - english-composition. I could add just its name to changelog, but when I was working on issue I had to check entire theme, so I decided to put cosmos name there, even it turned out that only one book is to fix.

Reply:
Ahh I understand, thanks for clarifying! :-)